src/Structure/Content/Shortcode.php

```diff
@@ -25,6 +25,22 @@ abstract class Shortcode implements ShortcodeInterface {
 	 */
 	protected $tag = '';
 
+	/**
+	 * The shortcode attributes when it is processed.
+	 *
+	 * @var array
+	 * @since 2020-01-31
+	 */
+	protected $attributes;
```

Comment:
Typically, in WordPress documentation and tutorials I find they use the variable `$atts` instead of `$attributes`.

When I initially tried to write my `render()` method it did not work for me because I used `$atts` (instead of the correct `$attributes`). Based on historical precedent (and the fact it is already burned into my brain), I'd prefer to see the property be `$atts`.

Reply:
I agree with Sal here, `$atts` is so ingrained in my mind that I messed that up when I first tried to test this on my local.

Reply:
I've changed this to $atts.